pages/about.tsx

```diff
@@ -15,139 +15,43 @@ export default createPage({
       <>
         <div className={style.aboutWrapper}>
           <div>
-            <p>
-              <b>What is the Good Governance Project [GGP]?</b>
-              <br />
-              <br />
-              We are a non-partisan, student-driven initiative at Boston College
-              Law School and the BC Clough Center for Constitutional Democracy.
-              We advocate for faith in democracy, and for democratic structures
+              <h1>Who we are</h1>
+              <p>
+              The Massachusetts Archive of Transparent Testimony platform is a project of the <a href="https://www.nulawlab.org">NuLawLab</a> developed with <a href="https://www.codeforboston.org">Code for Boston</a>. 
+              </p>
+              <p>
+              The NuLawLab is the interdisciplinary innovation laboratory at <a href="https://www.northeastern.edu/law/">Northeastern University School of Law</a>. NuLawLab's researchers are leaders in the emerging global Legal Design movement. 
```

Comment:
Consider adding a link to explain what Legal Design is (I don't know) or a more general blurb about the lab